subworkflows/local/genome_quant.nf

```diff
@@ -7,7 +7,7 @@ include { BOWTIE_ALIGN as BOWTIE_MAP_GENOME } from '../../modules/nf-core/bowtie
 
 workflow GENOME_QUANT {
     take:
-    ch_bowtie_index // channel: [genome.1.ebwt, genome.2.ebwt, genome.3.ebwt, genome.4.ebwt, genome.rev.1.ebwt, genome.rev.2.ebwt]
+    ch_bowtie_index // channel: [ val(meta), [ path(genome.1.ebwt), path(genome.2.ebwt), path(genome.3.ebwt), path(genome.4.ebwt), path(genome.rev.1.ebwt), path(genome.rev.2.ebwt) ] ]
```

Comment:
Sorry if I wasn't clear in my previous comment, but for this to work in `mirdeep2`  `ch_bowtie_index` should be of structure:

`channel: [ val(meta), [ path(one_directory)]` and not 
`channel: [ val(meta), [ path(genome.1.ebwt), path(genome.2.ebwt), path(genome.3.ebwt), path(genome.4.ebwt), path(genome.rev.1.ebwt), path(genome.rev.2.ebwt) ] ]`. 

Please write me if we need to review this together.

Reply:
This is something that is a bit more tricky to update, I created a new ticket to address it: https://github.com/nf-core/smrnaseq/issues/451